resources/views/components/cd/form/index.blade.php

```diff
@@ -0,0 +1,6 @@
+<form {{$attributes}}>
+    <fieldset class="{{$legendtype??'default'}}">  {{-- default or semantic --}}
+        <legend @if ($legend_sr_only??false == 'true') class="sr-only" @endif >{{$legend??'Form'}}</legend>
+        {{$slot}}
+    </fieldset>
```

Comment:
Why is a fieldset part of the base form? It seems like the features of the fieldset here might be better abstracted to their own component and then if you need a fieldset you can use it?

Reply:
I believe this is a WA consideration that is discussed in on Tone's form example page.

Reply:
Yep, I've seen what you mention. That's a good reason for fieldset to be in a component. But I found that when I went to try to use a fieldset within <x-cd.form> that it was treated as a nested fieldset and I couldn't avoid it.

I'm not sure where the balance on this lies between ease of use of the x-cd.form component versus needing to implement work-arounds when it isn't doing what you need.

Reply:
I don't think it is optional.

Reply:
Do you have a specific reference describing that a fieldset for a entire form is required? It is required when there is a group of related fields. But Tone's documentation encourages [not having nested fieldsets](https://iws-preview.hosting.cornell.edu/ama39/cssf/forms.html#section-6) and [avoiding wrapping a lot set of fields in a single fieldset](https://iws-preview.hosting.cornell.edu/ama39/cssf/forms.html#section-7). To follow these recommendations on anything other than simple forms, we would have to not use this base form component.